ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -972,16 +1162,10 @@ private void Visit(Cursor cursor, Cursor parent)
 
                 default:
                 {
-                    Debug.WriteLine($"Unhandled cursor kind: {cursor.KindSpelling} in {parent.KindSpelling}.");
-                    Debugger.Break();
+                    AddDiagnostic(DiagnosticLevel.Error, $"Unsupported cursor: '{cursor.KindSpelling}'. Generated bindings may be incomplete.", cursor);
```

Comment:
Places where we can't make any reasonable fallback and we will actually not end up parsing the whole tree are surfaced as errors.